easybuild/easyconfigs/x/XCfun/XCfun-20180122-intel-2017b-Python-2.7.14.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'MakeCp'
+
+name = 'XCfun'
+version = '20180122'
+commit = 'acd11bfbdcc2d2fb9c100111545bd479e6e2c452'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://dftlibs.org/xcfun/'
+description = """ XCFun is a library of DFT exchange-correlation (XC) functionals.
+ It is based on automatic differentiation and can therefore generate arbitrary order
+ derivatives of these functionals. """
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/dftlibs/xcfun/archive/']
+sources = ['%s.tar.gz' % commit]
```

Comment:
@smoors Please let EasyBuild rename the tarball on download (cfr. http://easybuild.readthedocs.io/en/latest/Writing_easyconfig_files.html#common-easyconfig-param-sources-alt)

```python
sources = [{'download_filename': '%s.tar.gz' % commit, 'filename': SOURCE_TAR_GZ}]
```